cluster/environments/azure-multiple-clusters-waf-tm-apimgmt/api-management.tf

```diff
@@ -0,0 +1,398 @@
+# resource "azurerm_resource_group" "api_mgmt_rg" {
```

Comment:
This file should be under cluster/azure, not cluster/environments ...

Reply:
Resolved and build test run successfully